cli/server/server.go

```diff
@@ -107,11 +108,23 @@ func getConfigFromContext(ctx *cli.Context) (config.Config, error) {
 	return config.Load(configPath, net)
 }
 
-// handleLoggingParams enables debugging output is that's requested by the user.
-func handleLoggingParams(ctx *cli.Context) {
+// handleLoggingParams enables debugging output and logPath requested by the user .
+func handleLoggingParams(ctx *cli.Context) error {
 	if ctx.Bool("debug") {
 		log.SetLevel(log.DebugLevel)
 	}
+
+	if logPath := ctx.String("log-path"); logPath != "" {
+		if err := io.MakeDirAndFile(logPath, "logger"); err != nil {
+			return err
+		}
+		f, err := os.OpenFile(logPath, os.O_WRONLY|os.O_CREATE, 0755)
```

Comment:
`0755` is certainly wrong here, but I'm also wondering whether it would be easier to use `os.Create`?

Reply:
done